From: Debian Games Team <pkg-games-devel@lists.alioth.debian.org>
Date: Tue, 26 Mar 2013 22:00:33 +0100
Subject: desktop-file

---
 FreeOrion/freeorion.desktop |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/FreeOrion/freeorion.desktop b/FreeOrion/freeorion.desktop
index 161cf57..817d453 100644
--- a/FreeOrion/freeorion.desktop
+++ b/FreeOrion/freeorion.desktop
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Name=FreeOrion
 GenericName=FreeOrion
-Comment=Run the FreeOrion client
+Comment=turn-based space empire and galactic conquest game (client)
 Exec=freeorion
 Icon=freeorion
 Terminal=true
 Type=Application
 Categories=Game;StrategyGame;
-URL=http://www.freeorion.org
\ No newline at end of file
+Keywords=space;strategy;empire;galaxy;turn-based;aliens;species;planets;fleet;research;resources;
