--- a/FreeOrion/CMakeLists.txt
+++ b/FreeOrion/CMakeLists.txt
@@ -52,9 +52,9 @@
 
 ########################################
 # Dependencies                         #
 ########################################
-set(Boost_USE_STATIC_LIBS ${BUILD_STATIC})
+set(Boost_USE_SHARED_LIBS ${BUILD_STATIC})
 set(Boost_USE_MULTITHREADED ${BUILD_MULTI_THREADED})
 find_package(Boost ${MINIMUM_BOOST_VERSION} COMPONENTS ${MINIMUM_BOOST_COMPONENTS})
 if (Boost_FOUND)
     include_directories(${Boost_INCLUDE_DIRS})
